src/main/scala/cognite/spark/v1/FileContentRelation.scala

```diff
@@ -42,7 +42,9 @@ class FileContentRelation(config: RelationConfig, fileExternalId: String, inferS
       backend <- Resource.make(
         IO(
           AsyncHttpClientFs2Backend
-            .usingClient[IO](SttpClientBackendFactory.create("file content download"), dispatcher))
+            .usingClient[IO](
+              SttpClientBackendFactory.create("file content download", Some(3600000)),
```

Comment:
maybe also accept customization via relation options

Reply:
Not sure what you mean, this backend is only used for file download.

Or do you mean in the SttpClientBackendFactory? I'm not sure that should be aware of the relation